code/modules/mob/living/carbon/human/species_types/ghoul.dm

```diff
@@ -0,0 +1,59 @@
+/datum/species/ghoul
+	name = "Ghoul"
+	id = "ghoul"
+	limbs_id = "ghoul"
+	inherent_traits = list(TRAIT_RADIMMUNE)
+	inherent_biotypes = list(MOB_INORGANIC, MOB_HUMANOID)
+	armor = -25 // very weak fag 100-25 = 85hp + the weak limbs is a big oof!
+	speedmod = 0.5 //little slow
+	burnmod = -1.25 // Leather skinless boys
+	brutemod = 1.2 //weaker fags
+	punchdamagehigh = 6
+	punchstunthreshold = 6
```

Comment:
given that their punches can range from 0 to 6 now instead of 0 to 9, and their stun threshhold is 6, this means that ghoul punches have a higher rate of stunning than human punches.

Reply:
Thanks adam! i never knew that